Zend/zend_language_scanner.l

```diff
@@ -1563,6 +1658,15 @@ NEWLINE ("\r"|"\n"|"\r\n")
 
 
 <ST_IN_SCRIPTING>"}" {
+	if(in_class) {
+		if(--in_class == 1) {
+			in_class = 0;	
+		}
+		if(in_use_list) {
+			in_use_list = 0;
+			in_trait_conflict = 0;
+		}
+	}
```

Comment:
Phew... ok, cases for `${` and `{$`  on class declaration context is closed. Indeed it was a easy fix.

Now moving to the "non qualified trait method name" case...